files/default/neutron-ha-tool.py

```diff
@@ -777,19 +786,19 @@ def wait_router_migrated(qclient, router_id, target_host, maxtries=60):
             remaining_ports = [
                 port['id'] for port in router_port_list['ports']
                 if (port['binding:vif_type'] != 'distributed' and
-                    not (port['status'] == 'ACTIVE' and
-                         port['binding:host_id'] == target_host))
+                    not (port['status'] == state and
+                         port['binding:host_id'] in ["", target_host]))
```

Comment:
What does it mean when `binding:host_id` is empty string?

Reply:
It means that a port is currently not bound to any host. On a second though it makes sense to have the empty string only for the `state=='DOWN'` case. In that case the `binding:host_id` can either be the "old" host or empty. For `state==ACTIVE` it should never be empty (TM) for non-distributed ports.

I'll rework that part a bit.

Reply:
> I'll rework that part a bit.

Done